docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/04 - Subprojects.md

```diff
@@ -0,0 +1,45 @@
+# Subprojects
+
+## Overview
+The **Subproject** plays an important role in determining the instrument battery that will be assigned to a candidate at a timepoint. **Subprojects** are defined in the `subproject` table of the database.
+
+## Adding Subproject Options
+
+### Front End (Recommended)
+Subprojects are defined in the Configuration module, which can be found in LORIS under the **Admin** menu tab.  Click on _To configure study subprojects click here_ link at the top of the page. Refer to the help section of the module for further instructions on how to add or modify subprojects.
+   
+### SQL
+Subprojects can be added directly in SQL using the following command.
+
+```sql
+INSERT INTO subproject (title) VALUES('SCI');
+```
+
+
+### API
+ _not yet available. See [API documentation](../../../API/) for latest additions_
+ 
+
+## Interaction With LORIS
+
+### Projects
+**Subprojects** must be associated to Projects in order to be able to create timepoints for candidates. This association should be defined directly on the front end through the configuration module.
+
+1. Navigate to the Configuration module
+2. In the upper left corner click on _To configure study projects click here._
+3. Under each project, you should be able to define the associated subprojects.
+
+> Note: If affiliation of subprojects to projects can not be done from the front-end, it could be done directly in SQL as follows. 
```

Comment:
This line is not great for 2 reasons: 
* This `if` statement obscures why the code snippet is here.  (like, can it be done in the front-end or not? we should know).  
* Again, the subproject(s) to project(s) relationship is fudged here. 
Better to state this as: 
```suggestion
> Here is an example of how to affiliate multiple subprojects to a single project with one back-end SQL command :
```

Reply:
i agree my formulation is not great but the suggestion makes it seem recommended to use SQL

i just wanna say in the event where you dont have access to the front end or you wanna do it in batch you can do that....

Reply:
great, then say it like that.  but the original phrasing wasn't clear as to what should be possible in Loris, particularly subproject(s):project(s) relationship

Reply:
done. please review